packages/components/src/components/marketplace-list.tsx

```diff
@@ -158,3 +130,99 @@ export const MarketplaceList = ({
     </section>
   );
 };
+
+export interface MarketplaceListItemProps extends React.ComponentPropsWithoutRef<'a'> {
+  item: IMarketplaceItemProps;
+}
+
+export function MarketplaceListItem({ item, ...rest }: MarketplaceListItemProps) {
+  return (
+    <Anchor
+      {...item.link}
+      {...rest}
+      className={cn(
+        'hive-focus flex gap-4 rounded-2xl border border-transparent bg-neutral-50 p-6 @container hover:border-neutral-200/50 hover:bg-neutral-100 @lg:gap-6 dark:bg-neutral-800 dark:hover:border-neutral-700 dark:hover:bg-neutral-700/50 [.green_&]:bg-green-900 [.green_&]:hover:border-green-700/50 [.green_&]:hover:bg-green-800/75',
+        item.link.className,
+      )}
+    >
+      <div
+        className={cn(
+          'size-16 shrink-0 rounded-lg bg-[--bg] @lg:size-16 @2xl:size-[92px] [.green_&]:[background:linear-gradient(135deg,_#68A8B6_0%,_#3B736A_100%)]',
+        )}
+      >
+        <Image
+          {...item.image}
+          placeholder={item.image.placeholder || 'empty'}
+          width="92"
+          height="92"
+          className="aspect-square rounded-lg object-contain"
+        />
+      </div>
+      <div className="flex flex-col">
+        <h3 className="m-0 line-clamp-2 font-medium text-[--fg] @lg:text-2xl">{item.title}</h3>
+        <div className="mb-2 line-clamp-3 text-sm text-[--fg-80] @lg:text-base">
+          {item.description}
+        </div>
+        {item.tags && item.tags.length > 0 && (
+          <TagsContainer className="mt-auto">
+            {item.tags.map(tagName => (
+              <Tag key={tagName}>{tagName}</Tag>
+            ))}
+          </TagsContainer>
+        )}
+        <div className="flex flex-wrap gap-x-4 text-xs text-[--fg-80] @lg:text-sm">
+          <span>
+            Updated <time dateTime={item.update}>{formatDate(item.update)}</time>
+          </span>
+          {item.weeklyNPMDownloads && (
+            <span className="hidden @sm:block">
+              {numberFormat.format(item.weeklyNPMDownloads)} weekly downloads
+            </span>
+          )}
+        </div>
+      </div>
+    </Anchor>
+  );
+}
+
+function moveFocusOnArrowKeys(event: React.KeyboardEvent<HTMLAnchorElement>, columns: number) {
+  let listItem: Element | null | undefined;
+
+  const move = ({ ArrowDown: '⬇', ArrowUp: '⬆', ArrowRight: '➡️', ArrowLeft: '⬅️' } as const)[
+    event.key
+  ];
+
+  if (!move) return;
+
+  if (move === '⬅️') {
```

Comment:
why not just use:
```suggestion
  if (move === 'ArrowLeft') {
```

because it looks differently on each system in text editor

Reply:
<img width="617" alt="image" src="https://github.com/user-attachments/assets/a70d924e-0772-47aa-bf93-7b39b4b4b29c">

Reply:
I wanted to do the early return in line 195. Would have to reorder the control flow a bit to skip the "move" variable.